getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -326,6 +397,158 @@ are constants:
         FIRE,
     }
 
+
+Code order
+----------
+
+This first section focuses on code order. For formatting, see
+:ref:`code_formatting`. For naming conventions, see :ref:`naming_conventions`.
+
+We suggest to organize GDScript code this way:
+
+::
+
```

Comment:
what about `tool` keyword?

Reply:
I'd put it at the top: it's important to know that a given piece of code will run in the editor, be it as part of a plugin or not. I don't mind exactly where, but I'd just put it on the first line.